src/RepoM.ActionMenu.Core/ActionMenu/Model/ActionMenus/BrowseRepository/RepositoryActionBrowseRepositoryV1Mapper.cs

```diff
@@ -50,14 +52,27 @@ protected override async IAsyncEnumerable<UserInterfaceRepositoryActionBase> Map
         }
     }
 
+    private static string try_get_remote_repo_long_name(string remoteUrl)
```

Comment:
I like that you are trying to solve/generate a remote name but i'm not sure this is the method I want